odev.js

```diff
@@ -27,10 +27,62 @@ import {createTableElements} from "./main.js";
 
 /* RESET ACTION */
 document.querySelector("#reset").addEventListener("click", () => {
-    createTableElements(data, "allcities");
-    createTableElements([], "singlecity")
+  createTableElements(data, "allcities");
+  createTableElements([], "singlecity");
 });
 
 /* START CODING HERE */
 
+document.querySelector("#populationBigger").addEventListener("click", () => {
+  const result = data.filter(city => city.population > 500000);
+  createTableElements(result, "allcities");
+  console.log('içerde: ', result);
+});
+
+document.querySelector("#landAreaLess").addEventListener("click", () => {
+  const result = data.filter(city => city.landArea < 1000);
+  createTableElements(result, "allcities");
+  console.log('içerde: ', result);
+});
+
+document.querySelector("#isPopulationLess").addEventListener("click", () => {
+  createTableElements([], "allcities");
```

Comment:
Bu satırda alert çıktıktan sonra boş array verdiğin için tablo sıfırlanıyor, tablo aynen dursa sadece alert çıksa da olurdu